recipes/nettle/all/conanfile.py

```diff
@@ -86,16 +88,27 @@ def _configure_autotools(self):
         else:
             conf_args.extend(["--disable-shared", "--enable-static"])
         self._autotools.configure(args=conf_args, configure_dir=self._source_subfolder)
+        # srcdir in unix path causes some troubles in asm files on Windows
+        if self.settings.os == "Windows":
+            tools.replace_in_file(os.path.join(self.build_folder, "config.m4"),
```

Comment:
This is because `m4` of cci is built with Visual Studio.
This makes it a pure windows executable.

When m4 is built with a cygwin/msys2 mingw compiler, the cygwin runtime is linked in too.
This runtime transparently converts unix paths to Windows paths.
Because "our" `m4` does not link to cygwin, it does not recognize unix paths.

Reply:
What a mess. So do you think that this fix is robust to all subystems?

Reply:
Your fix is fine (I think).
I think the runtime knows how to handle windows paths with `/` as separator.